src/rust/organization-management/src/client.rs

```diff
@@ -0,0 +1,98 @@
+use std::{
+    convert::TryInto,
+    time::Duration,
+};
+
+use rust_proto::organization_management::{
+    organization_management_service_client::OrganizationManagementServiceClient as _OrganizationManagementServiceClient,
```

Comment:
In other GRPC services, I believe we settled on a convention of naming things like this `FooProto`, rather than `_Foo`, because the leading underscore implies something that is not used.